Symfony/CS/Tests/ApplicationTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests;
+
+use Symfony\CS\Console\Application;
+
+/**
+ * @author SpacePossum
```

Comment:
Should be like this: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/c784f68e77052939f594044f8d30e7ce1b564543/Symfony/CS/Tests/IntegrationTest.php#L17.